### PR TITLE
Tests and Code Quality

### PR DESCRIPTION
**Pull request by ziyang27**

### CODE REVIEW

**Comment by ziyang27** [comment]
Code cov expected to drop

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 1]
```diff
@@ -26,24 +26,14 @@
 public class TypicalPersons {
 
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline")
-            .withAddress("123, Jurong West Ave 6, #08-111")
-            .withEmail("alice@example.com")
-            .withPhone("94351253")
-            .withTags("car")
-            .withUpcoming("01-05-2024 1500").build();
+            .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
+            .withPhone("94351253").withTags("car").withUpcoming("01-05-2024 1500").build();
```
Comment (Colex2000) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:30]: I think better change the upcoming year to 2025. 1st May might cause issue if someone tested this on 1st may or after.

**Review by Colex2000 — APPROVED** [review]
LGTM! Just one comment!

File: src/test/java/seedu/address/testutil/TypicalPersons.java (hunk 1 above)
Comment (ziyang27) [reply src/test/java/seedu/address/testutil/TypicalPersons.java:30]: Oh. This one is yc test, i tried changing the dates and it failed sth idk how to fix :(

Comment (tanyunchao) [reply src/test/java/seedu/address/testutil/TypicalPersons.java:30]: pass alr or still need me to fix @ziyang27

Comment (ziyang27) [reply src/test/java/seedu/address/testutil/TypicalPersons.java:30]: still cant, paiseh 😭

**Pull request merged**